### ISSUE TITLE
Form-side vs Route-side database operations

### ISSUE THREAD

**Issue by Pasmuth** [Pasmuth/hrdc-db#7]
I have figured out a way to make it so all forms can be entered from the form-side only, rather than from the route. You need to insert read-only fields for things like client_id or other variables that come from the route.

I managed to implement this strategy for the add address form. In forms.py -> CreateClientAddress() I added two integer fields:

`    created_by = IntegerField('User ID', validators = [DataRequired()])`
`    client_id = IntegerField('Client ID', validators = [DataRequired()])`

In the route I instantiate the form with those values already determined:

`    prefill = {'client_id':clientid,'created_by':current_user.id}`
`    form = CreateClientAddress(data = prefill)`

Then in the template I set those form fields to read-only:

`    {{ form.client_id.label }} {{ form.client_id(readonly=True) }}`
`   {{ form.created_by.label }} {{ form.created_by(readonly=True) }}`

This allows for the database operations associated with the form to be executed in the execute_transaction(self) function that lives inside the form.

TLDR: I need to update all the other forms to include this feature, so I can render all forms in the same way.

**Comment by Pasmuth** [comment]
I will close this issue when each of the old forms follows the new pattern.

~~-Add Address~~
-Add Contact Info
-Add Relationship
-Add Service

-Edit Client (this one may be more involved)
